recipes/pink/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pink" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/pin-{{ name }}/pin_{{ name }}-{{ version }}.tar.gz
```

Comment:
The PyPI name is pin-pink, right? What is the reason to rename it to just pink?

Reply:
This is related to the main dependency of this project, which is called Pinocchio but whose PyPI package is just `pin` (because `pinocchio` was already taken I guess).

The same happened with this project: it is called Pink, but there was already a package named `pink` on PyPI (which seems inactive). We then named the PyPI package `pin-pink`, adding the `pin` prefix in reference to Pinocchio.